src/component/organism/list/AnimalInfoList.js

```diff
@@ -2,8 +2,7 @@ import React from 'react';
 import {FlatList, ScrollView, Text, TouchableOpacity, View} from 'react-native';
 import {txt} from 'Root/config/textstyle';
 import AnimalInfo from 'Organism/listitem/AnimalInfo';
-import {animalInfoList} from 'Organism/style_organism';
-
+import {animalInfoList} from '../style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
```suggestion
import {animalInfoList} from 'Organism/style_organism copy';
```